backend/tests/api/test_main.py

```diff
@@ -97,21 +97,30 @@ def test_remove_food(self):
         assert response.status_code == 200
 
     def test_vendors_tray(self):
-        # Add tray_collect first
-        payload = {
+        # Add tray_collect
+        tray_collect_payload = {
             "pickup_time": "2020-10-08 11:11:51.291273",
             "vendor_id": 1,
         }
-        response = self.app.post("/add_tray_collect", json=payload)
+        self.app.post("/add_tray_collect", json=tray_collect_payload)
 
```

Comment:
ur code looks good!
but the three separate tests for the various tray components all have the same body content so it's a little hard to tell if its actually working properly without going into ur code every time.. i would just change that for later crud tests to make it easier to differentiate

Reply:
done!